fighters/dedede/src/acmd/aerials.rs

```diff
@@ -56,39 +56,66 @@ unsafe fn dedede_attack_air_f_game(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
     frame(lua_state, 1.0);
```

Comment:
Little bit of a nitpick but you can just get rid of the frame 1 stuff, no need to set the motion rate to 1.0 on frame 1

Reply:
Done!

Reply:
based